src/games/strategy/engine/data/GameSequence.java

```diff
@@ -14,7 +14,7 @@
   private int m_currentIndex;
   private int m_round = 1;
   private int m_roundOffset = 0;
-  private transient Object m_currentStepMutex = new Object();
+  private transient static final Object currentStepMutex = new Object();
```

Comment:
Static to protected against reflection not initializing this. There is only one GameSequence object, so sharing locks between multiple copies is okay.